partner_multi_relation_extended/models/res_partner_relation_all.py

```diff
@@ -73,3 +73,19 @@ def unlink(self):
                 "administrator can."
             ))
         return super(ResPartnerRelationAll, self).unlink()
+
+    @api.onchange('type_selection_id')
+    def onchange_type_selection_id(self):
+        res = super(ResPartnerRelationAll, self).onchange_type_selection_id()
+        if self.type_selection_id.type_id.is_work_relation:
+            res['warning'] = {
+                'title': _('Error!'),
```

Comment:
remove the "!"

Reply:
I finally used 'Warning' as a title, since it's not really an error.